rinad/configure.ac

```diff
@@ -169,7 +169,7 @@ AS_IF([test "$build_bindings_java" = "yes"],[
 AM_CONDITIONAL(BUILD_BINDINGS_JAVA, [test $build_bindings_java = yes])
 
 AC_PATH_PROG([MVN],[mvn],[])
-AS_IF([test x"$MVN" != x""],[
+AS_IF([test x"$MVN" = x""],[
```

Comment:
Hi Vincenzo,

Actually, when using test with AS_IF its supposed to be a single "=" for 
POSIX compliance.

If you still want it fixed no problem.
Regards,
Micheal



On 26/10/15 08:28, Vincenzo Maffione wrote:
>
> In rinad/configure.ac 
> <https://github.com/IRATI/stack/pull/774#discussion_r42968039>:
>
> > @@ -169,7 +169,7 @@ AS_IF([test "$build_bindings_java" = "yes"],[
> >  AM_CONDITIONAL(BUILD_BINDINGS_JAVA, [test $build_bindings_java = yes])
> >
> >  AC_PATH_PROG([MVN],[mvn],[])
> > -AS_IF([test x"$MVN" != x""],[
> > +AS_IF([test x"$MVN" = x""],[
>
> Please fix '=' into '=='
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/IRATI/stack/pull/774/files#r42968039>.
>


-- 
Micheal Crotty

PAS,
Telecommunications Software & Systems Group, Tel: +353 (0)51 302 939
ArcLabs Research and Innovation Building,    Fax: +353 (0)51 341 100
Waterford Institute of Technology,           Skype: mcrotty@tssg.org
Carriganore Campus, Carriganore,             www.tssg.org
Co. Waterford, Ireland